src/pages/dataTableUtilities/columns.js

```diff
@@ -15,6 +16,7 @@ const PAGE_COLUMN_WIDTHS = {
   stocktakes: [6, 2, 2, 1],
   stocktakeManager: [2, 6, 1],
   stocktakeEditor: [1, 2.8, 1.2, 1.2, 1, 0.8],
+  stocktakeEditorReasons: [1, 2.8, 1.2, 1.2, 1, 1, 0.8],
```

Comment:
need a common format for an altered but existing page -- `programSupplierRequisition`, `stocktakeEditorReasons` -- maybe a prefix `withX`

- `supplierRequisitionWithPrograms`
- `stocktakeEditorWithReasons`